webapp/src/app/ui/pull-request-widget/pull-request-widget.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, input } from '@angular/core';
+import { PullRequest } from '@app/core/modules/openapi';
+
+@Component({
+  selector: 'app-pull-request-widget',
```

Comment:
I don't like widget, we could call it `app-issue-card` because it can also be used for issues in the future and pull requests are a subclass of issues